### PR TITLE
[#199]preparing for 1.3.3-Hoxton

### PR DESCRIPTION
**Pull request by liubao68**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,11 +28,14 @@
...
-  <version>1.3.2</version>
+  <version>1.3.3-Hoxton-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Spring Cloud Huawei</name>
 
   <properties>
+    <java.version>1.8</java.version>
+    <spring-cloud.version>Greenwich.SR5</spring-cloud.version>
+    <spring-boot.version>2.1.6.RELEASE</spring-boot.version>
   </properties>
```
Comment (GuoYL123) [inline pom.xml:39]: need change to:
`<spring-cloud.version>Hoxton.SR3</spring-cloud.version>`
`<spring-boot.version>2.2.6.RELEASE</spring-boot.version>`

File: pom.xml [hunk 2]
```diff
@@ -28,11 +28,14 @@
 
   <groupId>com.huaweicloud</groupId>
   <artifactId>spring-cloud-huawei</artifactId>
-  <version>1.3.2</version>
+  <version>1.3.3-Hoxton-SNAPSHOT</version>
   <packaging>pom</packaging>
```
Comment (GuoYL123) [inline pom.xml:32]: is SNAPSHOT necessary？

File: pom.xml (hunk 1 above)
Comment (liubao68) [reply pom.xml:39]: will fix in next PR . This PR only change version.

File: pom.xml (hunk 2 above)
Comment (liubao68) [reply pom.xml:32]: use SNAPSHOT for developing, this is a convention and can reduce problems when user's execute `mvn deploy`

**Pull request merged**